### PR TITLE
MTP-1171_remove_x_from_multiselect

### PR DESCRIPTION
**Pull request by amundlindstol**

### CODE REVIEW

File: src/svelte/components/form/MultiSelect.svelte [hunk 1]
```diff
@@ -243,12 +245,6 @@
           ? `${selectId}-${activeOption.value}-${activeOptionIndex}`
           : undefined}
         {placeholder} />
-      <button
-        type="button"
-        class="button button--unstyled button--small remove-all icon-x"
```
Comment (lha047) [inline src/svelte/components/form/MultiSelect.svelte:248]: Maybe remove the `remove-all icon-x` style classes if not in use?

File: src/svelte/components/form/MultiSelect.svelte [hunk 2]
```diff
@@ -137,6 +134,8 @@
         add(activeOption)
       }
       inputValue = ''
+    } else if (e.keyCode === ESCAPE) {
```
Comment (lha047) [inline src/svelte/components/form/MultiSelect.svelte:137]: Nice 🌟

**Review by lha047 — CHANGES_REQUESTED** [review]
Good job :D 
Left some comments

File: src/svelte/components/form/MultiSelect.svelte [hunk 3]
```diff
@@ -150,8 +149,11 @@
     }
   }
 
-  function handleBlur(_): void {
-    optionsVisibility(false)
+  function handleBlur(e): void {
+    console.log(e.currentTarget)
```
[the remark this answers is not in the document]
Comment (amundlindstol) [reply src/svelte/components/form/MultiSelect.svelte:153]: ooups!

**Review by lha047 — APPROVED** [review]
Good job :D

**Pull request merged**